cpanfile.snapshot

```diff
@@ -2048,11 +2060,6 @@ DISTRIBUTIONS
       HTTP::Date 0
       Module::Build::Tiny 0.035
       perl 5.008001
-  HTTP-Lite-2.44
-    pathname: N/NE/NEILB/HTTP-Lite-2.44.tar.gz
-    provides:
-      HTTP::Lite 2.44
-    requirements:
```

Comment:
This is causing the Dockerless failure.

Reply:
I'm not entirely sure how to fix this, other than manually adding back that dependency?

Reply:
One really easy way to do this would be to blow away that commit and then go to the last passing Github action run for "Dockerless MetaCPAN". That has a build artifact which should include a `cpanfile.snapshot`. Overwrite your current snapshot with that one, commit it and you should be all set, since we know the tests pass with that generated snapshot. In this case, this should be the link to that build artifact: https://github.com/metacpan/metacpan-web/suites/1289931044/artifacts/20278695

Reply:
That seems to have worked, thanks for the guidance.